docs/content/users/install/ddev-installation.md

```diff
@@ -98,7 +98,7 @@ Docker or an alternative is required before anything will work with DDEV. This i
     10. Check that docker is working inside Ubuntu (or your distro): `docker ps`
     11. Optional: If you prefer to use the *traditional Windows* ddev instead of working inside WSL2, install it with `choco install -y ddev`. The Windows ddev works fine with the WSL2-based Docker engine. However, the WSL2 ddev setup is vastly preferable and at least 10 times as fast. Support for the traditional Windows approach will eventually be dropped.
     12. Open the WSL2 terminal, for example `Ubuntu` from the Windows start menu.
-    13. Install `ddev` with `curl -LO https://raw.githubusercontent.com/drud/ddev/master/scripts/install_ddev.sh && bash install_ddev.sh`
+    13. Install `ddev` with `curl -LsS https://raw.githubusercontent.com/drud/ddev/master/scripts/install_ddev.sh | sudo bash`
```

Comment:
Maybe `curl -LsS --fail` on these?

Reply:
Not sure, I think the user should see these errors, isn't it?

Reply:
Yes, the user should see it, but curl doesn't return a fail without that, and since it doesn't fail, the output often gets missed. Whatever :)

Reply:
Yes, looks better with the f option

![grafik](https://user-images.githubusercontent.com/25326036/179936115-94941001-4f20-4d3b-8099-124f20130d1d.png)